lib/app/Readme.md

```diff
@@ -8,12 +8,20 @@ In your configuration, you can enable an app using something like this:
 
 ```yaml
 apps:
-    # Replace appName with the name of the app you wish to enable
-    appName:
-        responseCache: 180
-        disabled: false
+  # Replace appName with the name of the app you wish to enable
+  appName:
+    responseCache: 180
+    disabled: false
+    sharedData:
```

Comment:
We always had this, but it was undocumented. I took the opportunity to clarify it. And because it was undocumented, I took the liberty to name it better (imho).